hw_printer_network/README.rst

```diff
@@ -0,0 +1,29 @@
+==========================
+ Hardware Network Printer
+==========================
+
+Hardware Network Printer
```

Comment:
{Put some short introduction first.}

{Then add more detailed description, technical specifications, any other information that could be interested for other developers. Don't forget that Usage instructions is a separated and has to be located in doc/index.rst file.}